other-docs/guides/updating-php/README.md

```diff
@@ -12,10 +12,10 @@ There are 2 key steps to getting ready for a new version of PHP:
 
 | Altis | PHP 8.0        | PHP 8.1       | PHP 8.2        | PHP 8.3        |
 |-------|----------------|---------------|----------------|----------------|
-| v21   |                | **Supported** | **Supported**  | *Experimental* |
-| v20   |                | **Supported** | **Supported**  | *Experimental* |
-| v19   |                | **Supported** | **Supported**  |                |
-| v18   | **Deprecated** | **Supported** | **Supported**  |                |
+| v22   | **Deprecated** | **Supported** | **Supported**  | **Supported**  |
```

Comment:
In this case, blank entry means not supported. So 8.0 has already passed from deprecated to not supported.
Given that no current Altis versions support it, just drop the whole 8.0 column.